.eslintrc.json

```diff
@@ -20,7 +20,10 @@
       "import/no-unresolved": "off",
       "no-shadow": "off",
       "no-useless-constructor": "off",
-      "react/prefer-stateless-function": "off"
+      "react/prefer-stateless-function": "off",
+      "react/no-unused-state": "off",
+      "react/prop-types": "off",
+      "react/destructuring-assignment": "off"
```

Comment:
[Optional] Please, always resist the urge to modify linter configuration files. They represent coding standards that you share with your team. This is marked optional because the previous reviewer had missed out on this requirement.

Reply:
@emmiiorji , these rules pass if we use functional components, but not class components, there is always a conflict of interest between the requirements we are given, and what the linters rules enforce, thus, we are left with little option.
if you hav some spare time, kindly look thru it and see it its possible to beat those rules with class type componets

Reply:
Hi @SSEKPIUS,

Thanks for your reply.

These are the rules you have turned off:
![image](https://user-images.githubusercontent.com/20554583/207806456-4c1488b5-a461-4da2-90ea-efcdc300ab55.png)


Going over them one after the other,

- The `useless-constructor` error can be avoided by including this assignment in the constructor `this.props = props;`. Fixing this will also fix the `prefer-stateless-function` error.
- The unused state error helps you avoid defining a state without using it
- For the prop-types error, you have to install prop-types. Check it's usage from [the documentation](https://www.npmjs.com/package/prop-types)
- For the destructuring-assignment error it prefers that you destructure an object e.g defining `{ next } = this.props.Result` and using the `next` variable instead of using `this.props.Result.next` directly

I hope this helps.

Reply:
Thanks a bunch @emmiiorji